addons/sourcemod/scripting/surftimer/commands.sp

```diff
@@ -4954,6 +4969,171 @@ public Action Command_CPR(int client, int args)
 	return Plugin_Handled;
 }
 
+public Action Command_PRinfo(int client, int args)
+{
+	if (!IsValidClient(client))
+		return Plugin_Handled;
+
+	switch(args){
+		//WORKS
+		//prinfo (in map zone) || (in bonus zone)
+		case 0:{
+			if(g_iClientInZone[client][2] == 0)
+				db_selectPRinfoUnknown(client, g_MapRank[client], 0, g_szSteamID[client]);
+			else
+				db_selectPRinfoUnknown(client, g_MapRankBonus[g_iClientInZone[client][2]][client], g_iClientInZone[client][2], g_szSteamID[client]);
+		}
+		//WORKS
+		//prinfo <mapname>
+		//prinfo <rank>
+		//prinfo <bonus number>
+		case 1:{
+			char arg1[128];
+			GetCmdArg(1, arg1, sizeof(arg1));
+
+			if (StrContains(arg1, "surf_", true) != -1)
+			{
+				db_viewPRinfoMapRank(client, g_szSteamID[client], arg1);
+			}
+			else if (StrContains(arg1, "@") != -1)
```

Comment:
I think `else if (arg1[0] = '@')` could work too and it's faster. ( @ESK0 😏 ) Also for other things like the `b` check